.github/workflows/tests.yml

```diff
@@ -0,0 +1,41 @@
+name: Run tests
+
+on:
+  pull_request:
+    branches:
+      - 'master'
+  push:
+    branches:
+      - 'master'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    services:
+      dynamodb:
+        image: amazon/dynamodb-local
+        ports: ["8000:8000"]
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Set up Ruby 2.6
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: 2.6.5
+
+    - name: Install Dynamo DB
+      run: |
+        sudo apt-get -yqq install libpq-dev
```

Comment:
Is libpq-dev required? Active Record isn't loaded in config/application.rb

Reply:
Ah great spot, I'll bin that whole line